katas/content/nonlocal_games/chsh_classical_win_condition/solution.md

```diff
@@ -0,0 +1,10 @@
+There are four input pairs (X, Y) possible, (0,0), (0,1), (1,0), and (1,1), each with 25% probability.
+In order to win, Alice and Bob have to output different bits if the input is (1,1), and same bits otherwise.
+
+To check whether the win condition holds, you need to compute $x ∧ y$ and $a ⊕ b$ and to compare these values: if they are equal, Alice and Bob won. [`Microsoft.Quantum.Logical`](https://docs.microsoft.com/qsharp/api/qsharp/microsoft.quantum.logical) library offers you logical functions `And` and `Xor` which you can use for this computation. Alternatively, you can compute these values using built-in operators: $x ∧ y$ as `x and y` and $a ⊕ b$ as `a != b`.
```

Comment:
The link to XOR function is added instead of current one.
Interesting, the classic CHSH doesn't need quantum logical functions but build-in operators. New link to Q# logical expression is added.

Reply:
The logical functions in the library effectively duplicate the built-in language operators. In classic QDK, defining functions made more sense, since they could sometimes be used as arguments to operations. In modern QDK, we don't need them, since in these (very rare) cases we can use partial application instead. XOR is effectively just "not equal" which is a built-in language operator, so I don't think we needed it either :-)

I'll take a look at the pull request a bit later.